Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF8 -Duser.language=en-US -Duser.country=US -Duser.language=en
com.google.javascript.jscomp.ExploitAssignsTest#testIssue1017
.E
Time: 3.55
There was 1 failure:
1) testIssue1017(com.google.javascript.jscomp.ExploitAssignsTest)
java.lang.RuntimeException: INTERNAL COMPILER ERROR.
Please report this problem.
INTERNAL COMPILER ERROR.
Please report this problem.
node is not a child
  Node(SCRIPT): testcode:1:33
x = x.parentNode.parentNode; x = x.parentNode.parentNode;
  Parent: NULL
	at com.google.javascript.rhino.Node.getChildBefore(Node.java:555)
	at com.google.javascript.rhino.Node.removeChild(Node.java:695)
	at com.google.javascript.jscomp.ExploitAssigns.collapseAssignEqualTo(ExploitAssigns.java:149)
	at com.google.javascript.jscomp.ExploitAssigns.collapseAssign(ExploitAssigns.java:53)
	at com.google.javascript.jscomp.ExploitAssigns.optimizeSubtree(ExploitAssigns.java:36)
	at com.google.javascript.jscomp.PeepholeOptimizationsPass$PeepCallback.visit(PeepholeOptimizationsPass.java:87)
	at com.google.javascript.jscomp.NodeTraversal.traverseBranch(NodeTraversal.java:540)
	at com.google.javascript.jscomp.NodeTraversal.traverseBranch(NodeTraversal.java:534)
	at com.google.javascript.jscomp.NodeTraversal.traverse(NodeTraversal.java:287)
	at com.google.javascript.jscomp.NodeTraversal.traverse(NodeTraversal.java:494)
	at com.google.javascript.jscomp.PeepholeOptimizationsPass$1.visit(PeepholeOptimizationsPass.java:71)
	at com.google.javascript.jscomp.NodeTraversal$1.shouldTraverse(NodeTraversal.java:480)
	at com.google.javascript.jscomp.NodeTraversal.traverseBranch(NodeTraversal.java:523)
	at com.google.javascript.jscomp.NodeTraversal.traverse(NodeTraversal.java:287)
	at com.google.javascript.jscomp.NodeTraversal.traverseChangedFunctions(NodeTraversal.java:485)
	at com.google.javascript.jscomp.PeepholeOptimizationsPass.process(PeepholeOptimizationsPass.java:63)
	at com.google.javascript.jscomp.CompilerTestCase.test(CompilerTestCase.java:845)
	at com.google.javascript.jscomp.CompilerTestCase.test(CompilerTestCase.java:477)
	at com.google.javascript.jscomp.CompilerTestCase.test(CompilerTestCase.java:403)
	at com.google.javascript.jscomp.CompilerTestCase.test(CompilerTestCase.java:372)
	at com.google.javascript.jscomp.CompilerTestCase.test(CompilerTestCase.java:360)
	at com.google.javascript.jscomp.CompilerTestCase.testSame(CompilerTestCase.java:599)
	at com.google.javascript.jscomp.ExploitAssignsTest.testIssue1017(ExploitAssignsTest.java:161)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at junit.framework.TestCase.runTest(TestCase.java:168)
	at junit.framework.TestCase.runBare(TestCase.java:134)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:157)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:136)
	at fr.inria.lille.repair.MethodTestRunner.runTest(MethodTestRunner.java:44)
	at fr.inria.lille.repair.MethodTestRunner.access$0(MethodTestRunner.java:37)
	at fr.inria.lille.repair.MethodTestRunner$1.call(MethodTestRunner.java:20)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.RuntimeException: INTERNAL COMPILER ERROR.
Please report this problem.
node is not a child
  Node(SCRIPT): testcode:1:33
x = x.parentNode.parentNode; x = x.parentNode.parentNode;
  Parent: NULL
	... 45 more
Caused by: java.lang.RuntimeException: node is not a child
	... 45 more

FAILURES!!!
Tests run: 1,  Failures: 1

com.google.javascript.jscomp.IntegrationTest#testExploitAssigns
.
Time: 0.116

OK (1 test)

com.google.javascript.jscomp.ExploitAssignsTest#testBug1840071
.E
Time: 2.325
There was 1 failure:
1) testBug1840071(com.google.javascript.jscomp.ExploitAssignsTest)
java.lang.RuntimeException: INTERNAL COMPILER ERROR.
Please report this problem.
INTERNAL COMPILER ERROR.
Please report this problem.
null
  Node(SCRIPT): testcode:1:15
a.b = a.x; if (a.x) {}
  Parent: NULL
	at com.google.javascript.rhino.Node.getChildBefore(Node.java:552)
	at com.google.javascript.rhino.Node.removeChild(Node.java:695)
	at com.google.javascript.jscomp.ExploitAssigns.collapseAssignEqualTo(ExploitAssigns.java:149)
	at com.google.javascript.jscomp.ExploitAssigns.collapseAssign(ExploitAssigns.java:53)
	at com.google.javascript.jscomp.ExploitAssigns.optimizeSubtree(ExploitAssigns.java:36)
	at com.google.javascript.jscomp.PeepholeOptimizationsPass$PeepCallback.visit(PeepholeOptimizationsPass.java:87)
	at com.google.javascript.jscomp.NodeTraversal.traverseBranch(NodeTraversal.java:540)
	at com.google.javascript.jscomp.NodeTraversal.traverseBranch(NodeTraversal.java:534)
	at com.google.javascript.jscomp.NodeTraversal.traverse(NodeTraversal.java:287)
	at com.google.javascript.jscomp.NodeTraversal.traverse(NodeTraversal.java:494)
	at com.google.javascript.jscomp.PeepholeOptimizationsPass$1.visit(PeepholeOptimizationsPass.java:71)
	at com.google.javascript.jscomp.NodeTraversal$1.shouldTraverse(NodeTraversal.java:480)
	at com.google.javascript.jscomp.NodeTraversal.traverseBranch(NodeTraversal.java:523)
	at com.google.javascript.jscomp.NodeTraversal.traverse(NodeTraversal.java:287)
	at com.google.javascript.jscomp.NodeTraversal.traverseChangedFunctions(NodeTraversal.java:485)
	at com.google.javascript.jscomp.PeepholeOptimizationsPass.process(PeepholeOptimizationsPass.java:63)
	at com.google.javascript.jscomp.CompilerTestCase.test(CompilerTestCase.java:845)
	at com.google.javascript.jscomp.CompilerTestCase.test(CompilerTestCase.java:477)
	at com.google.javascript.jscomp.CompilerTestCase.test(CompilerTestCase.java:403)
	at com.google.javascript.jscomp.CompilerTestCase.test(CompilerTestCase.java:372)
	at com.google.javascript.jscomp.CompilerTestCase.test(CompilerTestCase.java:360)
	at com.google.javascript.jscomp.ExploitAssignsTest.testBug1840071(ExploitAssignsTest.java:132)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at junit.framework.TestCase.runTest(TestCase.java:168)
	at junit.framework.TestCase.runBare(TestCase.java:134)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:157)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:136)
	at fr.inria.lille.repair.MethodTestRunner.runTest(MethodTestRunner.java:44)
	at fr.inria.lille.repair.MethodTestRunner.access$0(MethodTestRunner.java:37)
	at fr.inria.lille.repair.MethodTestRunner$1.call(MethodTestRunner.java:20)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.RuntimeException: INTERNAL COMPILER ERROR.
Please report this problem.
null
  Node(SCRIPT): testcode:1:15
a.b = a.x; if (a.x) {}
  Parent: NULL
	... 44 more
Caused by: java.lang.NullPointerException
	... 44 more

FAILURES!!!
Tests run: 1,  Failures: 1

com.google.javascript.jscomp.ExploitAssignsTest#testNestedExprExploitation
.E
Time: 2.347
There was 1 failure:
1) testNestedExprExploitation(com.google.javascript.jscomp.ExploitAssignsTest)
java.lang.RuntimeException: INTERNAL COMPILER ERROR.
Please report this problem.
INTERNAL COMPILER ERROR.
Please report this problem.
node is not a child
  Node(SCRIPT): testcode:1:18
a = 3; this.foo = a; this.bar = a; this.baz = 3;
  Parent: NULL
	at com.google.javascript.rhino.Node.getChildBefore(Node.java:555)
	at com.google.javascript.rhino.Node.removeChild(Node.java:695)
	at com.google.javascript.jscomp.ExploitAssigns.collapseAssignEqualTo(ExploitAssigns.java:149)
	at com.google.javascript.jscomp.ExploitAssigns.collapseAssign(ExploitAssigns.java:50)
	at com.google.javascript.jscomp.ExploitAssigns.optimizeSubtree(ExploitAssigns.java:36)
	at com.google.javascript.jscomp.PeepholeOptimizationsPass$PeepCallback.visit(PeepholeOptimizationsPass.java:87)
	at com.google.javascript.jscomp.NodeTraversal.traverseBranch(NodeTraversal.java:540)
	at com.google.javascript.jscomp.NodeTraversal.traverseBranch(NodeTraversal.java:534)
	at com.google.javascript.jscomp.NodeTraversal.traverse(NodeTraversal.java:287)
	at com.google.javascript.jscomp.NodeTraversal.traverse(NodeTraversal.java:494)
	at com.google.javascript.jscomp.PeepholeOptimizationsPass$1.visit(PeepholeOptimizationsPass.java:71)
	at com.google.javascript.jscomp.NodeTraversal$1.shouldTraverse(NodeTraversal.java:480)
	at com.google.javascript.jscomp.NodeTraversal.traverseBranch(NodeTraversal.java:523)
	at com.google.javascript.jscomp.NodeTraversal.traverse(NodeTraversal.java:287)
	at com.google.javascript.jscomp.NodeTraversal.traverseChangedFunctions(NodeTraversal.java:485)
	at com.google.javascript.jscomp.PeepholeOptimizationsPass.process(PeepholeOptimizationsPass.java:63)
	at com.google.javascript.jscomp.CompilerTestCase.test(CompilerTestCase.java:845)
	at com.google.javascript.jscomp.CompilerTestCase.test(CompilerTestCase.java:477)
	at com.google.javascript.jscomp.CompilerTestCase.test(CompilerTestCase.java:403)
	at com.google.javascript.jscomp.CompilerTestCase.test(CompilerTestCase.java:372)
	at com.google.javascript.jscomp.CompilerTestCase.test(CompilerTestCase.java:360)
	at com.google.javascript.jscomp.ExploitAssignsTest.testNestedExprExploitation(ExploitAssignsTest.java:108)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at junit.framework.TestCase.runTest(TestCase.java:168)
	at junit.framework.TestCase.runBare(TestCase.java:134)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:157)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:136)
	at fr.inria.lille.repair.MethodTestRunner.runTest(MethodTestRunner.java:44)
	at fr.inria.lille.repair.MethodTestRunner.access$0(MethodTestRunner.java:37)
	at fr.inria.lille.repair.MethodTestRunner$1.call(MethodTestRunner.java:20)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.RuntimeException: INTERNAL COMPILER ERROR.
Please report this problem.
node is not a child
  Node(SCRIPT): testcode:1:18
a = 3; this.foo = a; this.bar = a; this.baz = 3;
  Parent: NULL
	... 44 more
Caused by: java.lang.RuntimeException: node is not a child
	... 44 more

FAILURES!!!
Tests run: 1,  Failures: 1

com.google.javascript.jscomp.ExploitAssignsTest#testExprExploitation
.E
Time: 2.263
There was 1 failure:
1) testExprExploitation(com.google.javascript.jscomp.ExploitAssignsTest)
java.lang.RuntimeException: INTERNAL COMPILER ERROR.
Please report this problem.
INTERNAL COMPILER ERROR.
Please report this problem.
node is not a child
  Node(SCRIPT): testcode:1:28
a = null; b = null; var c = b
  Parent: NULL
	at com.google.javascript.rhino.Node.getChildBefore(Node.java:555)
	at com.google.javascript.rhino.Node.removeChild(Node.java:695)
	at com.google.javascript.jscomp.ExploitAssigns.collapseAssignEqualTo(ExploitAssigns.java:149)
	at com.google.javascript.jscomp.ExploitAssigns.collapseAssign(ExploitAssigns.java:50)
	at com.google.javascript.jscomp.ExploitAssigns.optimizeSubtree(ExploitAssigns.java:36)
	at com.google.javascript.jscomp.PeepholeOptimizationsPass$PeepCallback.visit(PeepholeOptimizationsPass.java:87)
	at com.google.javascript.jscomp.NodeTraversal.traverseBranch(NodeTraversal.java:540)
	at com.google.javascript.jscomp.NodeTraversal.traverseBranch(NodeTraversal.java:534)
	at com.google.javascript.jscomp.NodeTraversal.traverse(NodeTraversal.java:287)
	at com.google.javascript.jscomp.NodeTraversal.traverse(NodeTraversal.java:494)
	at com.google.javascript.jscomp.PeepholeOptimizationsPass$1.visit(PeepholeOptimizationsPass.java:71)
	at com.google.javascript.jscomp.NodeTraversal$1.shouldTraverse(NodeTraversal.java:480)
	at com.google.javascript.jscomp.NodeTraversal.traverseBranch(NodeTraversal.java:523)
	at com.google.javascript.jscomp.NodeTraversal.traverse(NodeTraversal.java:287)
	at com.google.javascript.jscomp.NodeTraversal.traverseChangedFunctions(NodeTraversal.java:485)
	at com.google.javascript.jscomp.PeepholeOptimizationsPass.process(PeepholeOptimizationsPass.java:63)
	at com.google.javascript.jscomp.CompilerTestCase.test(CompilerTestCase.java:845)
	at com.google.javascript.jscomp.CompilerTestCase.test(CompilerTestCase.java:477)
	at com.google.javascript.jscomp.CompilerTestCase.test(CompilerTestCase.java:403)
	at com.google.javascript.jscomp.CompilerTestCase.test(CompilerTestCase.java:372)
	at com.google.javascript.jscomp.CompilerTestCase.test(CompilerTestCase.java:360)
	at com.google.javascript.jscomp.ExploitAssignsTest.testExprExploitation(ExploitAssignsTest.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at junit.framework.TestCase.runTest(TestCase.java:168)
	at junit.framework.TestCase.runBare(TestCase.java:134)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:157)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:136)
	at fr.inria.lille.repair.MethodTestRunner.runTest(MethodTestRunner.java:44)
	at fr.inria.lille.repair.MethodTestRunner.access$0(MethodTestRunner.java:37)
	at fr.inria.lille.repair.MethodTestRunner$1.call(MethodTestRunner.java:20)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.RuntimeException: INTERNAL COMPILER ERROR.
Please report this problem.
node is not a child
  Node(SCRIPT): testcode:1:28
a = null; b = null; var c = b
  Parent: NULL
	... 44 more
Caused by: java.lang.RuntimeException: node is not a child
	... 44 more

FAILURES!!!
Tests run: 1,  Failures: 1

com.google.javascript.jscomp.IntegrationTest#testIssue550
.
Time: 0.123

OK (1 test)

com.google.javascript.jscomp.ExploitAssignsTest#testBug2072343
.E
Time: 2.251
There was 1 failure:
1) testBug2072343(com.google.javascript.jscomp.ExploitAssignsTest)
java.lang.RuntimeException: INTERNAL COMPILER ERROR.
Please report this problem.
INTERNAL COMPILER ERROR.
Please report this problem.
null
  Node(SCRIPT): testcode:1:12
a = a.x;a = a.x
  Parent: NULL
	at com.google.javascript.jscomp.NodeUtil.isExprAssign(NodeUtil.java:1621)
	at com.google.javascript.jscomp.ExploitAssigns.optimizeSubtree(ExploitAssigns.java:35)
	at com.google.javascript.jscomp.PeepholeOptimizationsPass$PeepCallback.visit(PeepholeOptimizationsPass.java:87)
	at com.google.javascript.jscomp.NodeTraversal.traverseBranch(NodeTraversal.java:540)
	at com.google.javascript.jscomp.NodeTraversal.traverseBranch(NodeTraversal.java:534)
	at com.google.javascript.jscomp.NodeTraversal.traverse(NodeTraversal.java:287)
	at com.google.javascript.jscomp.NodeTraversal.traverse(NodeTraversal.java:494)
	at com.google.javascript.jscomp.PeepholeOptimizationsPass$1.visit(PeepholeOptimizationsPass.java:71)
	at com.google.javascript.jscomp.NodeTraversal$1.shouldTraverse(NodeTraversal.java:480)
	at com.google.javascript.jscomp.NodeTraversal.traverseBranch(NodeTraversal.java:523)
	at com.google.javascript.jscomp.NodeTraversal.traverse(NodeTraversal.java:287)
	at com.google.javascript.jscomp.NodeTraversal.traverseChangedFunctions(NodeTraversal.java:485)
	at com.google.javascript.jscomp.PeepholeOptimizationsPass.process(PeepholeOptimizationsPass.java:63)
	at com.google.javascript.jscomp.CompilerTestCase.test(CompilerTestCase.java:845)
	at com.google.javascript.jscomp.CompilerTestCase.test(CompilerTestCase.java:477)
	at com.google.javascript.jscomp.CompilerTestCase.test(CompilerTestCase.java:403)
	at com.google.javascript.jscomp.CompilerTestCase.test(CompilerTestCase.java:372)
	at com.google.javascript.jscomp.CompilerTestCase.test(CompilerTestCase.java:360)
	at com.google.javascript.jscomp.CompilerTestCase.testSame(CompilerTestCase.java:599)
	at com.google.javascript.jscomp.ExploitAssignsTest.testBug2072343(ExploitAssignsTest.java:140)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at junit.framework.TestCase.runTest(TestCase.java:168)
	at junit.framework.TestCase.runBare(TestCase.java:134)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:157)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:136)
	at fr.inria.lille.repair.MethodTestRunner.runTest(MethodTestRunner.java:44)
	at fr.inria.lille.repair.MethodTestRunner.access$0(MethodTestRunner.java:37)
	at fr.inria.lille.repair.MethodTestRunner$1.call(MethodTestRunner.java:20)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.RuntimeException: INTERNAL COMPILER ERROR.
Please report this problem.
null
  Node(SCRIPT): testcode:1:12
a = a.x;a = a.x
  Parent: NULL
	... 42 more
Caused by: java.lang.NullPointerException
	... 42 more

FAILURES!!!
Tests run: 1,  Failures: 1

